src/net.cpp

```diff
@@ -781,7 +758,8 @@ static bool AttemptToEvictConnection(bool fPreferNewConnection) {
                 .nTimeConnected = node->nTimeConnected,
                 .nMinPingUsecTime = node->nMinPingUsecTime,
                 .addr = node->addr,
-                .nVersion = node -> nVersion
+                .vchKeyedNetGroup = node->vchKeyedNetGroup
+                .nVersion = node->nVersion,
```

Comment:
This doesn't compile. Squash this fix into 74bfd9a5c39e9fcd54416e3748f62f483f9b05b9:
```suggestion
                .vchKeyedNetGroup = node->vchKeyedNetGroup,
                .nVersion = node->nVersion
```